graphrag/index/workflows/v1/subflows/create_final_text_units.py

```diff
@@ -43,14 +38,11 @@ async def create_final_text_units(
     if final_covariates:
         final_covariates = cast(pd.DataFrame, final_covariates.table)
 
-    output = await create_final_text_units_flow(
+    output = create_final_text_units_flow(
```

Comment:
Is this not async anymore?

Reply:
No, the only async method inside it was the text_embed one so no async anymore.